routes/api.php

```diff
@@ -0,0 +1,6 @@
+<?php
+use Rapidez\ProductAlert\Http\Controllers\AlertController;
+
+Route::prefix('api')->group(function() {
+    Route::post('alerts', AlertController::class);
```

Comment:
Maybe something like `product-alerts` or `stock-alerts` may be a more suitable name for this endpoint as Magento also has Price alerts